chapters/en/module3.md

```diff
@@ -12,19 +12,708 @@ id: 3
+
+
+<choice id="1" >
+
+<opt text="True"  correct="true">
+
+You got it!  
+
+</opt>
+
+
+<opt text="False">
+
+We are usually interested in the values as a collective.
+
+</opt>
+
+
+</choice>
+
+**True or False**       
+*Layered histograms are more interpretable than stacked histograms.*
+
+<choice id="2" >
+
+<opt text="True"  >
+
+This isn't always true. In fact, depending on the question you are attempting to answer, layered histograms can do the opposite.
+
+</opt>
+
+<opt text="False" correct="true">
+
+You are on a roll!
+
+</opt>
+
+</choice>
+
+</exercise>
+
+
+<exercise id="3" title="Multple Choice Questions on Single Column Plots">
+
+**Question 1**      
+ Why can rugplots be problematic?
+
+<choice id="1" >
+<opt text="They can take a lot time to produce.">
+
+The time it takes to make a plot is more dependent on the amount of data you have!
+
+</opt>
+
+<opt text="They cannot visualize multiple features." >
+
+Rugplots are mostly used to display a single feature.  
+
+</opt>
+
+<opt text="They do not provide enough details for insights." >
+
+The problem might be that they have too much detail. 
+
+</opt>
+
+
+<opt text="With large quantities of data, They can be very difficult to interpret." correct="true">
+
+You got it!
+</opt>
+
+</choice>
+
+
+**Question 2**      
+To make a layered histogram instead of a stacked one, What should replace the `???` in the code below?
+
+
+```python
+hist_plot = alt.Chart(df_source).mark_bar(opacity=0.4).encode(
+    alt.X('runtime', bin=alt.Bin(maxbins=30)),
+    alt.Y('count()', ???),
+    color='genre')
+ .facet('country')
+ .resolve_scale(y='independent'))
+```
+
+
+
+<choice id="2" >
+<opt text="<code>stack=False</code>">
+
+You've got the right argument but the wrong value being set. 
+
+</opt>
+
+<opt text="<code>stack=None</code>" correct="true">
+
+Nice!
+
+</opt>
+
+<opt text="<code>layered=True</code>">
+
+I think your argument should be called something else.
+
+</opt>
+
+
+<opt text="<code>stacked=None</code>">
+
+You are extremely close, but no past tense is needed.
+
+</opt>
+
+</choice>
+
+
+</exercise>
+
+<exercise id="4" title="Stacked plotting with Penguins">
+
+
+**Instructions:**    
+Be patient when running a coding exercise for the first time, it can take a few minutes. 
+
+**When you see `____` in a coding exercise, replace it with what you assume to be the correct code. Run the code to see if you obtain the desired output
+and submit it to validate if you were correct.**
+
+
+Let's bring back our [penguins](https://www.kaggle.com/parulpandey/palmer-archipelago-antarctica-penguin-data) that we introduced in the previous module. 
+
+<codeblock id="penguins">
+
+</codeblock>
+
+
+For this question let's create a stacked histogram of the values in the `culmen_depth_mm` column for different penguin species.
+
+***([Culmen](https://allisonhorst.github.io/palmerpenguins/articles/articles/art.html): the upper ridge of a bird's beak)***
```

Comment:
I think we need to provide more context than this to make this interesting (and we need to, since I see this is used a lot in these exercises). So can we please include here (not linked to) an image of what the culmen is, and a sentence as to why we care about it? Does it vary across bird species and can be useful for predicting species type? Or is it indicative of the age/sex of an animal? etc?

Reply:
Added!